lib/commands/update.ts

```diff
@@ -15,49 +15,34 @@ export class UpdateCommand implements ICommand {
 		this.$projectData.initializeProjectData();
 	}
 
+	static readonly folders: string[] = ["lib", "hooks", "platforms", "node_modules"];
+	static readonly tempFolder: string = ".tmp_backup";
+
 	public async execute(args: string[]): Promise<void> {
-		const folders = ["lib", "hooks", "platforms", "node_modules"];
-		const tmpDir = path.join(this.$projectData.projectDir, ".tmp_backup");
+		const tmpDir = path.join(this.$projectData.projectDir, UpdateCommand.tempFolder);
 
 		try {
-			shelljs.rm("-fr", tmpDir);
-			shelljs.mkdir(tmpDir);
-			shelljs.cp(path.join(this.$projectData.projectDir, "package.json"), tmpDir);
-			for (const folder of folders) {
-				const folderToCopy = path.join(this.$projectData.projectDir, folder);
-				if (this.$fs.exists(folderToCopy)) {
-					shelljs.cp("-rf", folderToCopy, tmpDir);
-				}
-			}
+			this.backup(tmpDir);
```

Comment:
better API to understand is this.backup(source, destination), or if you use only source, why do you need a temp folder?

Reply:
I send the parameter just to avoid repetition of:
`const tmpDir = path.join(this.$projectData.projectDir, UpdateCommand.tempFolder);`